### ISSUE TITLE
When upgraded to GStreamer1.14, GStreamer-WARNING: Failed to load plugin

### ISSUE THREAD

**Issue by CCTV-1** [Alexpux/MINGW-packages#3623]
When I upgraded GStreamer to 1.14,all GStreamer Command line tools(gst-launch-1.0、gst-inspect-1.0 and other)  and using GStreamer program output:
`GStreamer-WARNING **: 09:59:04.514: Failed to load plugin 'D:\MSYS2\mingw64\lib\gstreamer-1.0\libgstvoamrwbenc.dll': 'D:\MSYS2\mingw64\lib\gstreamer-1.0\libgstvoamrwbenc.dll`
`GStreamer-WARNING **: 09:59:04.530: Failed to load plugin 'D:\MSYS2\mingw64\lib\gstreamer-1.0\libgstwebrtc.dll': 'D:\MSYS2\mingw64\lib\gstreamer-1.0\libgstwebrtc.dll`

`pacman -Ss GStreamer | grep installed` output:
```
mingw64/mingw-w64-x86_64-gst-libav 1.14.0-1 [installed]
mingw64/mingw-w64-x86_64-gst-plugins-bad 1.14.0-1 [installed]
mingw64/mingw-w64-x86_64-gst-plugins-base 1.14.0-1 [installed]
mingw64/mingw-w64-x86_64-gst-plugins-good 1.14.0-1 [installed]
mingw64/mingw-w64-x86_64-gst-plugins-ugly 1.14.0-1 [installed]
mingw64/mingw-w64-x86_64-gstreamer 1.14.0-1 [installed]
```

I updated to 1.14 in Fedora,everything is normal.i tried to reinstall `mingw-w64-x86_64-gst-plugins-bad`(libgstvoamrwbenc,libgstwebrtc in this package),but no effect.
there are too many packages in my msys2,now I don't have another computer to try to completely reinstall GStreamer.

**Comment by lazka** [comment]
Confirmed:

```
MISSING: C:/msys64/mingw32/lib/gstreamer-1.0/libgstvoamrwbenc.dll (gst-plugins-bad) -> libvo-amrwbenc-0.dll (vo-amrwbenc)
MISSING: C:/msys64/mingw32/lib/gstreamer-1.0/libgstwebrtc.dll (gst-plugins-bad) -> libnice-10.dll (libnice)
```

`pacman -S mingw-w64-x86_64-vo-amrwbenc mingw-w64-x86_64-libnice` should help

**Comment by lazka** [comment]
This was already fixed in #3583
